getting-started.md

```diff
@@ -758,76 +758,73 @@ In this section, we will provide an overview of the supply chain security use ca
 
 ### Sign: Introducing Image Signing & Verification to your Supply Chain
 
-**Overview**
+#### Overview
+
+This component allows a platform operator to define a policy that will
+restrict unsigned images from running on clusters.
+To enforce the configured policies this component communicates with external
+container registries to verify signatures on container images and make a
+decision based on the results of this verification. In order to make admission
+decisions this component is implemented as a dynamic admission control webhook.
 
-This feature-set allows an application operator to define a policy that will restrict unsigned images from running on clusters.
-This is done using a dynamic admission control component on Kubernetes clusters.
-This component contains logic to communicate with external registries and verify signatures on container images,
-making a decision based on the results of this verification.
 Currently, this component supports cosign signatures and its key formats.
-It will work with open source cosign, kpack and Tanzu Build Service (which is what we will overview in this document).
+Although this component does not sign container images, you could use tools such
+as the cosign CLI, kpack, and Tanzu Build Service (which is what we will overview
+in this document) to generate signatures for your images.
 
-Signing an artifact creates metadata about it that allows consumers to verify its origin and integrity.
-By verifying signatures on artifacts prior to their deployment,
-this allows operators to increase their confidence that trusted software is running on their clusters.
+Signing an artifact creates metadata about it that allows consumers to verify
+its origin and integrity.
+Operators can increase their confidence that trusted software is running on their
+clusters by verifying signatures on artifacts prior to their deployment.
 
-**Use Cases**
+#### Use Cases
 
 * Validate signatures from a given registry.
+* Deny unsigned or unverified images from being admitted in the cluster.
+
+> **Note**: this component does not verify images that are already running in a
+> cluster.
 
 **Signing Container Images**
 
-Tanzu Application Platform supports verifying container image signatures that follow the `cosign` format.
-Application operators may apply image signatures and store them in the registry in one of several ways:
+Tanzu Application Platform supports verifying container image signatures that
+follow the cosign format.
+Application operators may sign container images and store them in the registry
+in several different ways, including:
 
 * Using Tanzu Build Service v1.3
-
 * Using [kpack](https://github.com/pivotal/kpack/blob/main/docs/image.md#cosign-config) v0.4.0
-
 * Signing existing images with [cosign](https://github.com/sigstore/cosign#quick-start)
 
+**Supplying secrets for registries protected by authentication**
```

Comment:
Suggestion: It is commonly used the term `private registries` to refer to those with authentication.

Reply:
True, we changed from `private registries` to `registries protected by authentication` to be more clear on what we meant, but will change it back.